NuGetReleaseTool/NuGetReleaseTool/NuGetReleaseTool.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
```

Comment:
What is "value pos. 0" ?

![image](https://user-images.githubusercontent.com/1192347/199359307-01c6250b-5608-46d6-b6a8-da968f4dfa95.png)

Reply:
That's the argument, which is the release. 

Not sure why it shows as `value pos. 0`. 

Note that the behavior in question is existing, it existed pre-migration.